app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :likes
+  has_many :comments
```

Comment:
- [ ] the relation should have the **foriegn_key to author_id** like the ERD state